rules/explicitOverrideRule.ts

```diff
@@ -57,12 +59,13 @@ export class Rule extends Lint.Rules.TypedRule {
             * \`"${OPTION_JSDOC_TAG}"\` (default) Uses a jsdoc tag: \`/** @override */ method() { }\`
             * \`"${OPTION_EXCLUDE_INTERFACES}"\` Exclude interfaces from member override checks (default: false)
             * \`"${OPTION_FIX_PASCAL_CASE}"\` Uses PascalCase \`@Override\` for the jsdoc tag or decorator in the fixer (default: false)
+            * \`"${OPTION_BREAK_LINE_AFTER}"\` Breaks the line after jsdoc tag or decorator in the fixer (default: false)
         `,
         options: {
             type: 'array',
             items: {
                 type: 'string',
-                enum: [OPTION_DECORATOR, OPTION_JSDOC_TAG, OPTION_EXCLUDE_INTERFACES, OPTION_FIX_PASCAL_CASE],
+                enum: [OPTION_DECORATOR, OPTION_JSDOC_TAG, OPTION_EXCLUDE_INTERFACES, OPTION_FIX_PASCAL_CASE, OPTION_BREAK_LINE_AFTER],
```

Comment:
isn't "new line after decorator" a better name for this?

if yes, please replace everywhere

Reply:
it's not always the decorator, the fixer breaks after jsdoc tags too

Reply:
Should I change `OPTION_BREAK_LINE_AFTER` to `OPTION_NEW_LINE_AFTER` also?

Reply:
yeah, make the whole thing consistent

Reply:
fixed